eth2near/eth2near-block-relay-rs/src/eth2near_relay.rs

```diff
@@ -584,6 +585,21 @@ impl Eth2NearRelay {
 
             info!(target: "relay", "Sending light client update");
 
+            let finalized_block_number = return_on_fail!(
+                self.beacon_rpc_client
+                    .get_block_number_for_slot(types::Slot::new(
+                        light_client_update
+                            .finality_update
+                            .header_update
+                            .beacon_header
+                            .slot
+                            .as_u64()
+                    )),
+                "Fail on getting finalized block number"
+            );
+
+            info!(target: "relay", "Finalized block number = {}", finalized_block_number);
+
```

Comment:
This part should be after the information about light client update submission is happened. Otherwise, there's a chance that the relay will successfully send an update, but because of failing to get this finalized block from Beacon API RPC, we won't get this record in logs.

Reply:
I don't get you comment. Do you mean to change just the logs order or do you mean first of all submit the light client update and only after that get the finalized block for not failing on that operation? Or do you mean something else? https://github.com/aurora-is-near/rainbow-bridge/pull/818/commits/fa192e2debb67d173ed66462bf1763413946a12c ?

Reply:
Yes, to change the order, after we print logs for light client update, only then request & print logs for the finalized block number.

Reply:
fixed https://github.com/aurora-is-near/rainbow-bridge/commit/fa192e2debb67d173ed66462bf1763413946a12c